internal/codeintel/reposcheduler/options.go

```diff
@@ -0,0 +1,33 @@
+package reposcheduler
+
+import "time"
+
+type RepositoryBatchOptions struct {
+	// Time to wait before making repository visible again
+	// after previous scan. Note that this delay can be superseded if repository
+	// has been changed in the meantime
+	ProcessDelay time.Duration
+
+	// If allowGlobalPolicies is false, then configuration policies that define neither a repository id
+	// nor a non-empty set of repository patterns wl be ignored.
+	// When true, such policies apply over all repositories known to the instance.
+	AllowGlobalPolicies bool
```

Comment:
```suggestion
	// If AllowGlobalPolicies is false, then configuration policies that do not specify
	// a repository name or patterns will be ignored.
	// When true, such policies apply over all repositories known to the instance.
	AllowGlobalPolicies bool
```

Do we literally allow site admins to specify a repository id that is an integer? Or does "id" just here mean the base case of a literal pattern? If it's a string, might be clearer to say "repository name".